camunda-8-intermediate.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+// An action that deploys one or more resources (e.g. processes, decision models, or forms). This is an atomic call, i.e. either all resources are deployed or none of them are.
+async function deployResources() {
+  console.log("deploying resources");
+}
+
+// An action that creates and starts an instance of the specified process.
+async function createInstance() {
+  console.log(`creating process instance`);
+}
+
```

Comment:
@tmetzke Originally you requested we include "view process instance" -- do you mean [get process instance by its id](https://docs.camunda.io/docs/next/apis-tools/camunda-api-rest/specifications/get-process-instance/)? This is only in the 8.8 docs, so if you wanted this guide in the 8.7, perhaps we should change this up?

Reply:
We could add the "view process instance" in 8.8 additionally or simply leave it out entirely.

For 8.8, it would be a nice showcase of the search API that goes GA with 8.8, going to the secondary storage. We could shortly note the differences between the "create instance" (primary storage, engine interaction) and "view/get instance" (secondary storage, no engine interaction at all).

We also still have https://github.com/camunda/camunda/issues/26297 on the board to make this more visible and https://github.com/camunda/camunda-docs/pull/4580 to deliver an explanation for it that we could refer to from this tutorial. But it can also come later in an expansion issue for the tutorial when the other two are delivered (we still have capacity issues for those for the near future).

Reply:
Cool -- we can include this for 8.8 but leave out in the versions below.